swarm/services/swap/swap.go

```diff
@@ -102,12 +105,12 @@ func NewDefaultSwapParams() *SwapParams {
 	}
 }
 
-//this can only finally be set after all config options (file, cmd line, env vars)
-//have been evaluated
-func (self *SwapParams) Init(contract common.Address, prvkey *ecdsa.PrivateKey) {
+// Init this can only finally be set after all config options (file, cmd line, env vars)
+// have been evaluated
+func (localProfile *LocalProfile) Init(contract common.Address, prvkey *ecdsa.PrivateKey) {
```

Comment:
Thanks for fixing some of the linter problems and improving godoc!

In Go, when the context is small enough, you don't need to be very explicit with variable names - for example `localProfile` could very well just be `lp` in this scenario.

See: https://www.lysator.liu.se/c/pikestyle.html which is predecessor of many of the ideas for Go.

Reply:
Thanks for this feedback and resource, happy to make these changes and work to adapt this style into my future commits. 👍